apps/ui/assets/main.css

```diff
@@ -202,3 +202,13 @@ a[disabled] {
   border-left: 3px solid var(--bs-gray-500);
   padding: 2em 0 2em 3em;
 }
+
+.prism-editor__editor {
+  font-size: 16px;
+  font-family: monospace !important;
```

Comment:
Does it work without `!important`? If not, maybe it would be better to use an id instead of a class as CSS selector?